tests/basic/cr-comments-1-positive-out.yamlld

```diff
@@ -3,8 +3,7 @@
   "@type":
   - http://example.com/t1
   http://example.com/notacomment:
-  - http://example.com/this is#not:
-    - "@value": a comment
+  - {}  # That is what we get from `this is#not: a comment`. This behavior is confirmed by JSON-LD playground.
```

Comment:
I believe the original is correct, even if the playground gets a different value. "this is#not" should be treated as an IRI, relative to "http://example.com/". The fact that it is an illegal IRI is not something that the JSON-LD Expansion algorithms check for. But, arguably, we should have positive test results that contain illegal IRIs. It might be better to use something like `"this-is#not": a comment`.